tests/Schema/SchemaTest.php

```diff
@@ -228,7 +228,10 @@ public function testHasTableForQuotedAsset(): void
 
     public function testHasNamespace(): void
     {
-        $schema = new Schema();
+        $schemaConfig = new SchemaConfig();
+        $schemaConfig->setName('public');
```

Comment:
This and other similarly modified tests operate unqualified and qualified names. From now on, this is only allowed if the schema has a default namespace configuration.